recipes-kernel/linux/linux-raspberrypi-dev.bb

```diff
@@ -5,19 +5,27 @@ python __anonymous() {
         raise bb.parse.SkipRecipe(msg)
 }
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi:"
+LINUX_VERSION ?= "5.10.y"
+LINUX_RPI_BRANCH ?= "rpi-5.10.y"
 
-LINUX_VERSION ?= "4.19"
-LINUX_RPI_BRANCH ?= "rpi-4.19.y"
+SRCREV_machine = "${AUTOREV}"
+SRCREV_meta = "${AUTOREV}"
```

Comment:
will AUTOREV be processed during parsing? we need something like linux-yocto-dev to ignore it.

Reply:
The anonymous python at the top of the file ensures the recipe is ignored if not selected as the preferred kernel provider.